doc/whatsnew/2.13.rst

```diff
@@ -29,6 +29,13 @@ Extensions
     Docstring classification is now based on the highest amount of matched sections instead
     of the order in which the docstring styles were tried.
 
+* ``DocStringStyleChecker``
+
+    * Disable checker ``bad-docstring-quotes`` for Python <= 3.7, because in this versions the line
+      numbers decorated functions and classes are not reliable
```

Comment:
```suggestion
    * Disable checker ``bad-docstring-quotes`` for Python <= 3.7, because in these versions the line
      numbers for decorated functions and classes are not reliable which interferes with the checker.
```